cfme/containers/node.py

```diff
@@ -27,16 +27,17 @@
 resource_locator = "//div[@id='records_div']/table//span[@title='{}']"
 
 
-class NodeView(BaseLoggedInPage):
-    title = Text('#explorer_title_text')
+class NodeView(ContainerObjectAllBaseView):
+    TITLE_TEXT = 'Nodes'
 
     monitor = Dropdown('Monitoring')
-    policy = Dropdown('Policy')
-    download = Dropdown('Download')
-
     nodes = Table(locator="//div[@id='list_grid']//table")
 
     @property
+    def table(self):
```

Comment:
**REQUIRED**: Any reason to add a convenience here? I would prefer to just leave it as nodes and modify things that are already looking at ```table``` to point to ```nodes``` instead.